tests/e2e/conftest.py

```diff
@@ -458,11 +458,13 @@ def neuro_rm_dir(
     )
 
 
-def neuro_ps(timeout: int) -> t.Set[str]:
+def neuro_status(job_id: str, timeout: int) -> str:
     out = run(
-        f"neuro --quiet ps",
+        f"neuro status {job_id}",
         timeout_s=timeout,
-        debug=True,
+        debug=False,
         stop_patterns=DEFAULT_NEURO_ERROR_PATTERNS,
     )
-    return set(out.split())
+    search = re.search(r"Status: (\w+)", out)
+    assert search, f"not found job status in output: `{out}`"
+    return search.group(1)
```

Comment:
I feel like returning `True/False` would be more convenient. It feels weird to compare return value of `neuro_status` with `succeeded` — what does it imply to a person reading the code?

Reply:
done thanks!